### ISSUE TITLE
Template Already installed when trying to import after removal

### ISSUE THREAD

**Issue by porter9992** [BiglySoftware/BiglyBT#3105]
I had jackett running, but I have been installing and uninstalling the template. Now it just says it is already installed.  

I've attached pictures & Debug.

Java 21.0.1 (64 bit)
  Oracle Corporation
e:\program files\openjdk

SWT v4956r12, win32, zoom=100, dpi=96
Windows 10 v10.0, amd64 (64 bit)
B3.5.0.0/4 az3
-----------------
Jackett JSON template: `{"vuze":{"components":[{"content":{"web.time_zone":"GMT-6","web.required_cookies":[],"name":"Jackett","select_rec":1,"uid":"\xd88b5ff4189272385eaf\x","last_updated":1548157139579,"az_version":5,"version":3,"selected":2,"source":2,"id":2929201020,"l1_map":[{"from":6,"to":6,"maps":[]}],"l2_map":[{"from":6,"to":8,"maps":[]}],"web.search_url_format":"http://127.0.0.1:9117/api/v2.0/indexers/all/results?apikey=za7xgbsm7ctxemvjhb7jk5z4j9k9i36b&Query=%s&_=1548156611274","web.needs_auth":0,"web.auto_date":1,"json.path":"Results","rank_bias":"1.0","web.maps":[{"field":6,"name":"CategoryDesc"},{"field":1,"name":"Title"},{"field":102,"name":"MagnetUri"},{"field":102,"name":"Link"},{"field":103,"name":"Details"},{"field":2,"name":"PublishDate"},{"field":3,"name":"Size"},{"field":5,"name":"Seeders"},{"field":4,"name":"Peers"},{"field":200,"name":"InfoHash"}],"pref_count":"0.0","web.full_cookies":null,"type":2,"web.auth_method":"transparent"},"type":1}]}}`
-----------------
Win 10 Pro 22H2
BiglyBT 3.5
OpenJDK 21.0.1

![Jackett Template](https://github.com/BiglySoftware/BiglyBT/assets/150206533/ebcfc791-fcd9-4324-b7bc-c7ac03e7c405)
![Jackett Already Installed](https://github.com/BiglySoftware/BiglyBT/assets/150206533/d71408cb-f872-4ea4-8587-0bdc8ff5d4da)
[debug.zip](https://github.com/BiglySoftware/BiglyBT/files/13454868/debug.zip)

**Comment by parg** [comment]
Weird, something must have got confused. If you do a "search" by using the search box at the top of BiglyBT you'll get the search results panel. There is an "add/edit templates" button in the top area, to the left of the "create subscription" button. Hit that and you'll get a new window to add/edit sites. Is Jackett listed there? If so then delete it using the little 'x' button and save the results.

If this doesn't work then edit the JSON string that you pasted above to change the "id". It current has

`... "id":2929201020 ...`

chang that number, it just needs to be unique amongst search templates

You might also want to change the "name" field in case you need to distinguish between this one and the half-installed one.

**Comment by porter9992** [comment]
Nope, it doesn't show there either.

Yes, I tried renaming but I didn't realize about the ID number, but it took it.  I'll remove the new one and re-add with a different number, just in-case that is why there is such a long response time.
-------------------------
I just incremented the ID by 1 and now I get no dialog at all when I try to import _(I of course tried closing Bigly and reopening and then importing)_.  So I changed the name and incremented again to a new # and that showed up.  

I searched and found the name in the Bigly config file, but I don't exactly see what and how to remove that appropriately. 


Not too much of a problem since it can get to work by not reusing a name that was used before.
